src/CommunityToolkit.Maui.MediaElement/Views/MauiMediaElement.macios.cs

```diff
@@ -13,16 +15,15 @@ public class MauiMediaElement : UIView
 	/// Initializes a new instance of the <see cref="MauiMediaElement"/> class.
 	/// </summary>
 	/// <param name="playerViewController">The <see cref="AVPlayerViewController"/> that acts as the platform media player.</param>
-	/// <param name="parentViewController">The <see cref="UIViewController"/> that acts as the parent for <paramref name="playerViewController"/>.</param>
 	/// <exception cref="NullReferenceException">Thrown when <paramref name="playerViewController"/><c>.View</c> is <see langword="null"/>.</exception>
-	public MauiMediaElement(AVPlayerViewController playerViewController, UIViewController? parentViewController)
+	public MauiMediaElement(AVPlayerViewController playerViewController)
```

Comment:
just to point out that this's a breaking change

Reply:
@pictos  can you give me an example?

Reply:
There might be a better solution: #1623 
@pictos if you can look at that PR and get back to me if that is a better solution that does not cause breaking changes? Someone else is working on a related problem that may make this PR moot.

Reply:
@ne0rrmatrix I'm not saying this is wrong or bad... Just point out that's a breaking change, so if this PR gets merged we will need to release a major version. What you can do is preserve the old ctor and mark it with the Obsolete attribute and not show it on Intellisense.

Reply:
Ok i will restore old ctor and mark obsolete attribute.

Reply:
@pictos  I reverted ctor and marked old ctor as ```[Obsolete]``` Hope I did it right. Please review and suggest any changes.